src/tests.builds

```diff
@@ -6,6 +6,10 @@
     <GenerateCodeCoverageReportForAll>true</GenerateCodeCoverageReportForAll>
   </PropertyGroup>
 
+  <PropertyGroup Condition="'$(BuildingUAPVertical)'=='true' and '$(SkipTests)' != 'true' " >
```

Comment:
Since what we run is actually an app, you can't run tests in parallel locally, because we install the test app and run it, and then uninstall it again for every test library. It's not the best way to do it, I agree, but I don't think anybody would be running all of the uap tests on one machine.